saskatoon/harvest/signals.py

```diff
@@ -37,6 +39,46 @@ def changed_by(sender, instance, **kwargs):
     else:
         instance.changed_by = None
 
+def notify_pending_status_update(sender, instance, **kwargs):
+    # Send email only if pending status is removed
+    if instance.id:
+        original_instance = sender.objects.get(id=instance.id)
+        if original_instance.pending and not instance.pending:
+            property_owner_email = list()
+            if instance.owner:
+                contact_person = instance.owner.get_person()
+                if not contact_person:
+                    try:
+                        contact_person = (
+                            instance.owner.get_organization().contact_person
+                        )
+                    except:
+                        raise RuntimeError("Property owner is neither a person nor an organization")
```

Comment:
There are three unknowns properties that will make this block execute If I understand correctly.
I’d suggest we simply return None instead of crashing.